### ISSUE TITLE
Final drive: network drive

### ISSUE THREAD

**Issue by Kar7upeli5** [MrPig91/PSChiaPlotter#63]
How to add final drive if i use network drives? I trying map network drive on plotting PC, but no results :(

**Comment by MrPig91** [comment]
Can you tell me which version of PSChiaPlotter you have installed? You can get this by running the command below.
Import-Module PSChiaPlotter
Get-Module PSChiaPlotter

**Comment by Kar7upeli5** [comment]
ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     1.0.27     PSChiaPlotter                       {ConvertTo-FriendlyTimeSpan, Get-ChiaHarvesterActivity, Ge...

**Comment by Kar7upeli5** [comment]
1.0.27

**Comment by MrPig91** [comment]
Hmm, it should show up with that version. Does the network drive have a drive letter associated with it?
Does the following command output anything for you?
Get-CimInstance -ClassName Win32_MappedLogicalDisk

**Comment by Kar7upeli5** [comment]
1: Yes - O:\ P:\ R:\...
2: Nothing

**Comment by MrPig91** [comment]
Thank you for that information. What OS are using, are there more than one person signed into the computer per chance? Either way, I might take a different approach of grabbing network drives. Can you let me know if this grabs the network drives that are missing. 

Get-CimInstance -Class Win32_LogicalDisk -Filter "DriveType=4"

**Comment by Kar7upeli5** [comment]
All computers have Win 10 Pro. The user is myself, but I am connected via a remote desktop connection. I tried directly but also without results

**Comment by MrPig91** [comment]
Did "Get-CimInstance -Class Win32_LogicalDisk -Filter "DriveType=4"" produce any results?

**Comment by Kar7upeli5** [comment]
Nothing

Maybe the problem is permissions?

**Comment by Kar7upeli5** [comment]
So, after command "net use O: \\server\share" i can see my network folder ;)

**Comment by Mavyre** [comment]
> So, after command "net use O: \server\share" i can see my network folder ;)

Perfectly what I needed, thank you!

**Comment by Kar7upeli5** [comment]
Today i have another problem PSChiaPlotter can't see disk array drives attached to my server. All drives is listed after command Get-PSDrive, but PSChiaPlotter does not see the attached drives

**Comment by MrPig91** [comment]
I have just release a new version, can you see if it sees them with the newest version?

**Comment by Kar7upeli5** [comment]
Cool
Now I update the module and everything works fine

**Comment by MrPig91** [comment]
Whew, let me know if something does not work correctly. I changed more than I should have in one release, but there was a lot of feedback that I wanted to address. Thank you so much for taking the time to give me feedback on this module, you helped make it better.
